Changed some things in the Preparation section, more soon (#2)
Miepee/AM2R-Modding-Wiki: content/utmt/rooms/creating new rooms/_index.md

  - name: optimalRoomSide
    src: "optimalRoomSide.png"
    title: Optimal placing for left and right transitions
  - name: RoomSetup
    src: "RoomSetup.png"

>     title: This is how it should look now

Reviewer: IMO "This is approximately how the room should look after the preparation" sounds+looks better as an alt

  - name: RoomOnCreate
    src: "RoomOnCreate.png"
    title: This is how it will look when first adding a new room
  - name: ViewsSetup
    src: "ViewsSetup.png"

>     title: If you've done everything correctly this is how it should look 

Reviewer: This is how the Views should look like.

> ---

Reviewer: Personally, I like camelCase for assets, so `roomOnCreate` instead of `RoomOnCreate`. For everyones sake who wants to contribute, the assets should all have a similar naming style.

TODO: having some more screenshots in here would be nice. Also fix some phrasing.

### Preparation:

> Before we set up tiles or textures, we must first make some preperations, to get started, create a new room by right clicking on `Rooms` in the file viewer on the left and then click add. Now you will see 3 things; 

Reviewer: ```suggestion
Before we set up tiles or textures, we must first make some preperations. To get started, create a new room by right-clicking on `Rooms` in the viewer on the left and then click `add`. Now you will see 3 things; 
```

> A list of all the ojects that are in your room

Reviewer: ```suggestion
A list of all objects that are in your room.
```

> A set of values, corresponding to the object in your room that you currently have selected

Reviewer: ```suggestion
A set of values, corresponding to the element that you currently have selected.
```
Since values apply to backgrounds/views/room name etc. too

> A Visual overview of your room, if you're adding a new room, this will be completely black with a grey grid.

Reviewer: Use lists for these please

> A list of all the ojects that are in your room
> A set of values, corresponding to the object in your room that you currently have selected
> A Visual overview of your room, if you're adding a new room, this will be completely black with a grey grid.

Reviewer: ```suggestion
A visual overview of your room. If you're adding a new room, this will be completely black with a grey grid.
```

{{< img name="RoomOnCreate" size=origin >}}

> When creating a new room, make __absolutely__ sure the dimensions are multiples of `320` x `240`. Having different room dimensions will cause lots of troubles. Also make sure that the room's room speed is set to `60` as that's the Framerate what AM2R runs at. Entering different values will lead to the room being slowed-down or sped up.  

Reviewer: ```suggestion
When creating a new room, make __absolutely__ sure the dimensions are multiples of `320` x `240`. Having different room dimensions will cause lots of troubles. Also make sure that the room's room speed (`Speed`) is set to `60` as that's the framerate AM2R runs at. Entering different values will lead to the room being slowed-down or sped up.  
```

> Once you have your set room size, set a background by searching for `bgScreenGuide` and putting it in one of the background definitions by dragging and dropping it from the left. Enable the background afterwards.  

Reviewer: ```suggestion
Once you have your set room size, set a background by searching for `bgScreenGuide` and drag-and-droppig it into one of the background definitions. Enable the background afterwards.  
```

> You probably also want to give your room a name, for this, it is recommended to follow the naming convention of all the other rooms in the game, meaning that you start with `rm_` then the area your room belongs to (`a0`, `a1`, `a2`, etc..), a letter to denote which subsection of that area you are in (`h` for outside, and then `a`, `b`, etc...) and finally the number of the room itself (`01`, `02`, `03`, etc...) 

Reviewer: ```suggestion
You probably also want to give your room a name. For this, it is recommended to follow the naming convention of all the other rooms in the game: start with `rm_`, then the area your room belongs to (`a0`, `a1`, `a2`, etc..), a letter to denote which subsection of that area you are in (`h` for outside, and then `a`, `b`, etc.) and finally the number of the room itself (`01`, `02`, `03`, etc...).
```

> e. g. If you were adding a Room inside Golden Temple, you would name it something like `rm_a1a13` 

Reviewer: ```suggestion
If you were adding a room inside Golden Temple for example, you would name it something like `rm_a1a13`. But of course you can use any names you like.  
```

Reviewer: Don't forget to use two spaces at the end of a line to indicate a newline in markdown.

> Note that UTMT does not sort things alphabetically, but instead always puts the newest addition at the bottom of the folder

Reviewer: ```suggestion
Note that UTMT does not sort things alphabetically. Instead it always puts the newest addition at the bottom of the list.
```

  - name: viewsSetup
    src: "viewsSetup.png"
    title: This is how the Views should look like.
  - name: sceenGuideSetup
    src: "sceenGuideSetup.png"
    title: This is what is should like after setting up the bgScreenGuide Element 
---

TODO: having some more screenshots in here would be nice. Also fix some phrasing.

## Preparation
Before we set up tiles or textures, we must first make some preperations. To get started, create a new room by right-clicking on `Rooms` in the viewer on the left and then click `add`. Now you will see 3 things;
- A list of all objects that are in your room.
- A set of values, corresponding to the element that you currently have selected.
- A Visual overview of your room. If you're adding a new room, this will be completely black with a grey grid.

{{< img name="roomOnCreate" size="small" >}}

When creating a new room, make __absolutely__ sure the dimensions are multiples of `320` x `240`. Having different room dimensions will cause lots of troubles. Also make sure that the room's room speed (`Speed`) is set to `60` as that's the framerate AM2R runs at. Entering different values will lead to the room being slowed-down or sped up.  

> You probably also want to give your room a name. For this, it is recommended to follow the naming convention of all the other rooms in the game: start with `rm_`, then the area your room belongs to (`a0`, `a1`, `a2`, etc..), a letter to denote which subsection of that area you are in (`h` for outside, and then `a`, `b`, etc.) and finally the number of the room itself (`01`, `02`, `03`, etc...).  

Reviewer: ```suggestion
You probably also want to give your room a name. For this, it is recommended to follow the naming convention of all the other rooms in the game: start with `rm_`, then the area your room belongs to (`a0`, `a1`, `a2`, etc.), a letter to denote which subsection of that area you are in (`h` for outside, and then `a`, `b`, etc.) and finally the number of the room itself (`01`, `02`, `03`, etc...).  
```
